lib/webapp.js

```diff
@@ -710,6 +728,26 @@ const staticHandlers = {
     return router;
   },
 
+  swagger(productCode) {
+    const router = express.Router();
+    const agentSwaggerPlugin = zluxUtil.agentSwaggerPlugin;
+    const agentSwaggerBaseUrl = `${zLuxUrl.makePluginURL(productCode, agentSwaggerPlugin)}/catalogs/swagger`;
+    const zluxSwaggerBaseUrl = `${zLuxUrl.makePluginURL(productCode, 'org.zowe.configjs')}/catalogs/swagger`;
```

Comment:
configjs isnt zlux. its a nearly essential plugin but it is an independent plugin.
you could make another proxy plugin, `org.zowe.zlux`, but proxying the server to itself might cause strange bugs, so i think its best to just skip that.

Reply:
created separate plugin, would like you to try if its even working